src/utils/daos_hdlr.c

```diff
@@ -3072,20 +3067,19 @@ dm_disconnect(struct cmd_args_s *ap,
 	      struct file_dfs *dst_file_dfs)
 {
 	int rc = 0;
+	int rc2;
 
 	if (src_file_dfs->type == DAOS) {
 		if (is_posix_copy) {
 			rc = dfs_umount(src_file_dfs->dfs);
 			if (rc != 0) {
-				fprintf(ap->errstream, "failed to unmount "
-				"source (%d)\n", rc);
+				fprintf(ap->errstream, "failed to unmount source (%d)\n", rc);
 				D_GOTO(out, rc);
```

Comment:
From dfs_umount
```suggestion
				D_GOTO(out, daos_der2errno(rc));
```

Reply:
Good spot, I thought I'd got these.

Other than the logging of source/destination I did wonder if this function would be best with a helper function which is simply called once with src and once with dest.